### ISSUE TITLE
Installation problems

### ISSUE THREAD

**Issue by ayalpani** [excalidraw/mermaid-to-excalidraw#25]
Hi, I have been waiting long for mermaid to arrive in excalidraw and I have picked it up immediately after your tweet @ad1992 @excalibot 

it seems the provided examples only work if I install the @next version of the excalidraw dependency or the `convertToExcalidrawElements` export will not be available:

```
  "dependencies": {
    "@excalidraw/excalidraw": "^0.15.2-d73cd14",
    "@excalidraw/mermaid-to-excalidraw": "^0.1.0",
```

Also the module resolution does not seem to work. Inside the `dist` folder I had to create myself a quickfix by renaming 

- `constants.js` to `constants`
- `graphToExcalidraw.js` to `graphToExcalidraw`
- `interfaces.js` to `interfaces`
- `parseMermaid.js` to `parseMermaid`
- `utils.js` to `utils`

Also I believe this example has an error:

https://docs.excalidraw.com/docs/@excalidraw/mermaid-to-excalidraw/installation

In it you `await parseMermaid` but shouldn't it be `await convertToExcalidrawElements` ?

**Comment by ad1992** [comment]
@ayalpani yes you are right currently the mermaid version depends on `@next` version as the API `convertToExcalidrawElements` isn't shipped to stable version, however we are going to release the stable version this week, post that `@next` version will be replaced with stable version.

As per the example, `convertToExcalidrawElements` isn't async so we don't need `await` syntax for it. Are you facing any error with the example ?

You can also refer to [`mermaid integration PR in Excalidraw`](https://github.com/excalidraw/excalidraw/pull/6920/files) as a reference if you are trying to integrate this library into your application

**Comment by ad1992** [comment]
The version is now upgraded to the stable version `0.16.0`
As per the file path, this seems to be the behavior of tsc as it doesn't append `.js` extension when compiling to `js`

https://github.com/microsoft/TypeScript/issues/42151 and https://github.com/microsoft/TypeScript/issues/16577 so looks like the way to fix this would be to use `.js` extension in our code itself.

**Comment by ad1992** [comment]
The file path issue should be fixed by #30

**Comment by ad1992** [comment]
closed by #30 and #28
